swap_meet/vendor.py

```diff
@@ -0,0 +1,102 @@
+import sys
+
+class Vendor:
+    # wave 01
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory == None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        for i, thing in enumerate(self.inventory):
+            if thing == item:
+                return self.inventory.pop(i)
+
+        return False
+
+    # wave 02
+    def get_by_category(self, category):
+        items = []
+
+        for item in self.inventory:
+            if item.category == category:
+                items.append(item)
+
+        return items
+    
+    #  wave 03
+    def swap_items(self, friend, my_item, their_item):
+        if len(self.inventory) == 0 or len(friend.inventory) == 0:
```

Comment:
This could be a little simpler.  This works because `[]` is falsey.

```suggestion
        if self.inventory or friend.inventory:
```